input/pagecontent/reference.md

```diff
@@ -593,6 +595,14 @@ Since slices are sublists, a sliced array path technically points to the *first*
   component[respirationScore][fiveMinuteScore].code
   ```
 
+* Paths to the MedicationRequest and NuritionOrder reference options in an "orders" slice that allows both types:
```

Comment:
Small typo in "NutritionOrder" here

Reply:
Fixed and rebased into https://github.com/HL7/fhir-shorthand/pull/177/commits/98504e62b3c3d8f6a905bf7315067f117190b3f1.